templates/web/common/site_layout.tt.html

```diff
@@ -367,7 +367,7 @@ <h3 class="title-5 text-medium">[% lang('footer_join_the_community') %]</h3>
 						<p>[% f_lang('f_join_us_on_slack', { url => "https://slack.openfoodfacts.org" } ) %]</p>
 						<p><a href="https://forum.openfoodfacts.org/">Forum</a></p>
 						<p id="footer_social_icons">[% f_lang('f_footer_follow_us_links', { links => '
-							<a href="https://twitter.com/' _ twitter_account _ '"><img src="/images/icons/dist/twitter.svg" class="footer_social_icon" alt="Twitter"></a>
+							<a href="https://x.com/OpenFoodFacts' _ X_account _ '"><img src="/images/icons/dist/X_logo_2023.svg" class="footer_social_icon" alt="X"></a>
```

Comment:
```
You need to rename in common.pot, and en.po
#. make sure the Twitter account exists for your country. Otherwise, ask @stephane to create it
--
69 | msgctxt "twitter_account"
70 | msgid "OpenFoodFacts"
71 | msgstr ""


```

Reply:
@teolemon  changes done in common.pot, and en.po 
Are these changes satisfactory?